promql/printer_test.go

```diff
@@ -63,6 +63,12 @@ func TestExprString(t *testing.T) {
 		{
 			in: `sum(task:errors:rate10s{job="s"}) BY (code)`,
 		},
+		{
+			in: `sum(task:errors:rate10s{job="s"}) WITHOUT ()`,
+		},
+		{
+			in: `sum(task:errors:rate10s{job="s"}) WITHOUT (code)`,
```

Comment:
this duplicates the test below it

Reply:
Oh whoops, didn't notice. Removed my second test.